### PR TITLE
added configurations for kind for the pseudo-named section in quick start

### PR DESCRIPTION
**Pull request by Nikita-Levuskin**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Other (please describe): add configs for quick start

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

### CODE REVIEW

**Comment by Jorres** [comment]
Could you please elaborate, who needs these configurations?

If it's for running tests locally, they WILL fail on such a config because kind with more than 1 node is required, YDB test installation will simply fail to schedule.

If it's just for launching a single node locally:
- you don't need extra mounts (https://github.com/ydb-platform/ydb-kubernetes-operator/pull/254/files#diff-ebfe2e1d359a9ae70980a6f0e7fb56b6dc8fd62f40dddef8dce55024ee9301f4R10)
- add some explanation on why\how to run such a configuration, for example, here: https://github.com/ydb-platform/ydb-kubernetes-operator/tree/master/docs. If you don't mention it in the docs, the only person who will use these configs is going to be just you.

File: samples/kind/storage.yaml [hunk 1]
```diff
@@ -0,0 +1,91 @@
+apiVersion: ydb.tech/v1alpha1
+kind: Storage
+metadata:
+  name: storage-kind-sample
+spec:
+  monitoring:
```
Comment (Jorres) [inline samples/kind/storage.yaml:6]: Isn't the disabled monitoring the default?

Comment (Nikita-Levuskin) [reply samples/kind/storage.yaml]: fixed

**Comment by Jorres** [comment]
Discarded due to a more complete PR with the same features merged: https://github.com/ydb-platform/ydb-kubernetes-operator/pull/258

**Pull request closed without merging**